pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -35,11 +35,12 @@ stdenv.mkDerivation {
   '';
 
   buildPhase = ''
-    node scripts/ninja.js build
+    # This is an unfortunate name, but it's actually how to build a release
+    # binary for BuckleScript
```

Comment:
isn't `BS_RELEASE_BUILD` environment variable enough?

Reply:
I don't think we technically need it, the build script sets is: https://github.com/BuckleScript/bucklescript/blob/0c44d04ea08a4748376de020b2ea76b1ba622981/scripts/install.js#L29